go/integTest/setup_teardown_test.go

```diff
@@ -0,0 +1,120 @@
+// Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0
+
+package integTest
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+)
+
+type GlideTestSuite struct {
+	suite.Suite
+	standalonePorts []int
+	clusterPorts    []int
+	redisVersion    string
+}
+
+func (suite *GlideTestSuite) SetupSuite() {
+	// Stop cluster in case previous test run was interrupted or crashed and didn't stop.
+	// If an error occurs, we ignore it in case the servers actually were stopped before running this.
+	runClusterManager(suite, []string{"stop", "--prefix", "redis-cluster"}, true)
+
+	// Delete dirs if stop failed due to https://github.com/aws/glide-for-redis/issues/849
+	err := os.RemoveAll("../../utils/clusters")
+	if err != nil && !os.IsNotExist(err) {
+		log.Fatal(err)
+	}
+
+	// Start standalone Redis instance
+	clusterManagerOutput := runClusterManager(suite, []string{"start", "-r", "0"}, false)
+
+	suite.standalonePorts = extractPorts(suite, clusterManagerOutput)
+	suite.T().Logf("Standalone ports = %s", fmt.Sprint(suite.standalonePorts))
```

Comment:
Sorry for stupid golang question - what does `suite.T()` do? Where it is defined?

Reply:
For tests that aren't part of a test suite, you need to pass a paramater of type testing.T which provides a test context and methods for printing and logging (more info [here](https://go.dev/doc/tutorial/add-a-test)). Here we are using a test suite, and you access testing.T via suite.T(). This is mentioned in Testify's docs [here](https://pkg.go.dev/github.com/stretchr/testify/suite#Suite.T)